src/server/memory/stream_store.h

```diff
@@ -86,6 +88,7 @@ class StreamStore {
   std::shared_ptr<BulkStore> store_;
   size_t threshold_;
   std::unordered_map<ObjectID, std::shared_ptr<StreamHolder>> streams_;
+  std::unordered_map<ObjectID, int64_t> stream_marks_;
```

Comment:
Could we use the `mark` as an attribute of `StreamHolder`? without introduce an extra map. Considering that when deleting a stream, we need an extra deletion and may lead to inconsistency.